nextcloud_update.sh

```diff
@@ -152,6 +152,13 @@ download_script MENU menu
 # Make sure fetch_lib.sh is available
 download_script STATIC fetch_lib
 
+# Get newest dat files for geoblock.sh
+if grep -q "^#Geoip-block" /etc/apache2/apache2.conf
+then
+    # get_newest_dat_files # TODO: Uncomment this in a followup PR to be able to test this properly
```

Comment:
this will be uncommented in a followup PR.

Reply:
Move this to the adminer update section please.

Reply:
done